binder/src/main/java/io/grpc/binder/SecurityPolicies.java

```diff
@@ -116,6 +135,39 @@ public Status checkAuthorization(int uid) {
     };
   }
 
+  /**
+   * Creates {@link SecurityPolicy} which checks if the SHA-256 hash of the package signature
+   * matches any of {@code requiredSignatureSha256Hashes}.
+   *
+   * @param packageName the package name of the allowed package.
+   * @param requiredSignatureSha256Hashes the SHA-256 digests of the signatures of the allowed
+   *     package.
+   * @throws NullPointerException if any of the inputs are {@code null}.
+   * @throws IllegalArgumentException if {@code requiredSignatureSha256Hashes} is empty, or if any
+   *     of the {@code requiredSignatureSha256Hashes} are not of length 32.
+   */
+  public static SecurityPolicy oneOfSignatureSha256Hash(
+      PackageManager packageManager,
+      String packageName,
+      ImmutableList<byte[]> requiredSignatureSha256Hashes) {
```

Comment:
s/ImmutableList/List/

We shouldn't expose Guava on the public API surface. You can make a "copy" using ImmutableList and if the original list was ImmutableList it becomes a noop. However, my other comment about byte[] means this doesn't really matter.

Reply:
Yeah, that's what I was suggesting. Although `Collection` is a bit confusing instead of `List` since `Set<byte[]>` would be broken/useless since `byte[]` uses reference equality. I'd probably use `Iterable<byte[]>` instead of `Collection<byte[]>`. To me it probably doesn't matter much between choosing `List<byte[]>` and `Iterable<byte[]>`.